code/ReportAdmin.php

```diff
@@ -227,28 +239,60 @@ public function getEditForm($id = null, $fields = null)
             $fields = $report->getCMSFields();
         } else {
             // List all reports
-            $fields = new FieldList();
+            $fields = FieldList::create();
             $gridFieldConfig = GridFieldConfig::create()->addComponents(
+                // This is a container component that is required by filter header component
+                GridFieldButtonRow::create('before'),
+                $filterHeader = GridFieldFilterHeader::create(),
                 GridFieldSortableHeader::create(),
-                GridFieldDataColumns::create(),
-                GridFieldFooter::create()
+                $columns = GridFieldDataColumns::create(),
+                GridFieldPageCount::create(),
+                GridFieldPaginator::create()
             );
+
+            // Configure the filter header filter search form
+            $generalField = BasicSearchContext::config()->get('general_search_field_name');
+            $searchFieldList = FieldList::create([
+                HiddenField::create($generalField),
+                TextField::create('Title'),
+                TextField::create('Description'),
+            ]);
+
+            $searchContext = BasicSearchContext::create(Report::class);
+            $searchContext->setFields($searchFieldList);
+
+            // Setup filter configuration - partial match with case-insensitive modifier
+            $filters = [
+                'Title',
+                'Description',
+            ];
+
+            foreach ($filters as $fieldName) {
+                $fieldFilter = PartialMatchFilter::create($fieldName);
+                $fieldFilter->setModifiers([
+                    // We want case-insensitive match to be consistent with other areas of the CMS
+                    'nocase',
+                ]);
```

Comment:
Case sensitivity is based on configuration that can change per project. We shouldn't be enforcing a specific case sensitivity here.
```suggestion
```

Reply:
At a higher level, or do you mean in general?

This specific situation doesn't make sense to not be this way though.